### PR TITLE
New addon: Disable warning not save (2)

### PR DESCRIPTION
**Pull request by valmontechno**
I formatted the code with Prettier compared to my last pull request.

Disables warning when exiting or refreshing an unsaved project.

### CODE REVIEW

**Comment by TheColaber** [comment]
When would this be useful?

**Comment by valmontechno** [comment]
This is useful with the addon that disables autosave to test simple code without saving it and to use only the costume editor.

File: addons/disable-warning-not-save/addon.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Disable warning not save",
```
Comment (Norbiros) [inline addons/disable-warning-not-save/addon.json:2]: Maybe we can change name a little bit?
For example `Disable warning about closing not saved project` or something like that.

Comment (Samq64) [reply addons/disable-warning-not-save/addon.json:2]: `Disable leave editor warning`?

**Comment by Norbiros** [comment]
@valmontechno Can you enable actions on your fork [here](https://github.com/valmontechno/ScratchAddons/actions)?

**Comment by apple502j** [comment]
Too dangerous IMO.

**Comment by apple502j** [comment]
@WorldLanguages Can you say "rejected"

**Comment by BroJac5246** [comment]
It could be marked "dangerous," but who would use it?

**Comment by WorldLanguages** [comment]
Rejected until we decide otherwise on an issue.

**Comment by WorldLanguages** [comment]
@valmontechno If you really want this to be added, create an issue and explain the motivation in detail

**Pull request closed without merging**